helpers/misc-utils.ts

```diff
@@ -19,6 +19,28 @@ export const setDRE = (_DRE: HardhatRuntimeEnvironment) => {
   DRE = _DRE;
 };
 
+export const isForkNetwork = (): boolean => {
+  return process.env.FORK ? true : false;
```

Comment:
improvement: it's possible to use `!!` or `Boolean` to convert string value to boolean.

Reply:
ok, but I just find it more readable this way